core/src/test/java/org/apache/calcite/rel/metadata/BlankMetadataHandler.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to you under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.calcite.rel.metadata;
+
+/**
+ * A blank {@link MetadataHandler} that is used as a base for adding a synthetic method.
+ */
+interface BlankMetadataHandler extends MetadataHandler<TestMetadata> {
```

Comment:
Can you move the test interfaces into the test class files?

Reply:
Moved `BlankMetadataHandler` into a test class.

`TestMetadata` and `MetadataHandlerWithStaticMethod` are used by two distinct test classes, so they cannot be 'internalized' without fusing the test classes together. If your suggestion is to still fuse them, please elaborate what test structure you would prefer to see.